benchmarks/ig-en/jw300-baseline/README.md

```diff
@@ -0,0 +1,62 @@
+# IGBO TO ENGLISH
+**Authors:**
+Dino Anastasopoulos (1900661)
+Jesse Bristow (1875955)
+Chloe Smith (1877342)
+
+## DATA
+```sh
+The JW300 Igbo-English dataset
+```
+
+## Model
+
+Transformer model (Using JoeyNMT)
+
+Link  to model:
+
+```sh
+https://drive.google.com/drive/folders/1tAWvmFu2eiuWqjOfThXaGrYqBI0Yx01U?usp=sharing
```

Comment:
When I tried to access it, I got the following message: "This folder is in the owner's bin. To view this folder, ask the owner to restore it." - Would you please restore it or move it to somewhere else? If it's too large, you can go ahead and delete model checkpoints, and only keep the best/last one.